Cargo.toml

```diff
@@ -55,6 +55,7 @@ rustdoc-args = ["--cfg", "doc_cfg"]
 [workspace]
 members = [
         ".",
-        "examples/postgres/pooled-with-rustls"
+        "examples/postgres/pooled-with-rustls",
+        "examples/postgres/run-pending-migrations-with-rustls",
```

Comment:
Can you enable the new example in the CI as well: https://github.com/weiznich/diesel_async/blob/e3c3511c3db1191ed26c6a68f7e719ec7840e289/.github/workflows/ci.yml#L120 ?